modules/ROOT/pages/deployment/general/roles-permissions.adoc

```diff
@@ -0,0 +1,128 @@
+* Admins and Space Admins can currently only view and manage spaces they are member of.
+* Users can't create spaces or set quotas for spaces they manage.
+
+Over time, the web front-end will be updated to provide capabilities in the best and most user friendly way.
+
+=== Permission Granularity
+
+Compared to ownCloud Server where e.g. the administrator has more or less full access to everything, Infinite Scale refines roles with assigned permissions types and particular permissions for that type on a much more detailed level. This is not only true for system roles abut also for spaces roles.
+
+=== Security
+
+With role and permission separation (system/spaces/sharing), the overall security is greatly increased and security standards like *separating tasks and data access by design* is addressed.
+
+== System Roles and Permissions
+
+Currently there are four predefined system roles with fixed permissions assigned to specific roles. Note that some permissions are only granted via the role for the particular user but not covering all users. A defined role can be assigned to a user, concatenating roles is currently not supported.
+
+Consider that these roles and permissions are provided by the system via APIs, which means you can call them via own code or `curl`. 
+
+Read the image below the following way::
+* A Role gets one or more permission types assigned.
+* Each permission type is granted individual rights relevant for that role.
+* Roles are assigned to users.
+
+// the empty macro is necessary to have more distance between text and image
+ 
+{empty} +
+
+// the table references following ocis code where roles and permissions a re defined: https://github.com/owncloud/ocis/blob/master/services/settings/pkg/store/defaults/defaults.go
+
+image::deployment/general/roles-and-permissions-matrix.svg[width=550]
+
+=== System Role Details
+
+In general with regards to spaces:
+
+* Spaces can always be deleted by the those who are granted managing the space with the exception of the `Delete Home Space` permission, which is only assigned to the Admin for the reason when a user account gets deleted.
+* Managing a space means creating or deleting a space, setting quotas, delegating the right to manage space members and space member roles, but not having access to the content of the space except when being a member of the space.
+* When creating a space, the creator is automatically in the delegatable Manager role for this space. For details see xref:space-roles-and-permissions[Space Roles and Permissions].
+
+Admin::
+The `Admin` role is in full control of all permission types with one exception which is `Self Management`. This permission type is excluded to avoid that the last admin accidentally reduces their own role to a lower one making the system unmanageable. As long as there is more than one admin, roles can be set by the `Role Management` permission.
```

Comment:
```suggestion
The `Admin` role provides full control of all permission types with one exception, which is `Self Management`. This permission type is excluded to avoid that the last remaining admin accidentally reduces their own role to a lower one making the system unmanageable. As long as there is more than one admin, roles can be set via the `Role Management` permission.
```